### PR TITLE
Release 3.3.2

### PR DESCRIPTION
**Pull request by FinalAngel**
- documentation improvements
- fix an issue with ``setStorage`` and ``getStorage`` on private browsing mode
- freeze ``package.json`` and ``bower.json`` versions

### CODE REVIEW

File: static/js/addons/cl.utils.js [hunk 1]
```diff
@@ -95,8 +100,29 @@ var Cl = window.Cl || {};
...
+            }
+        },
+
+        /**
+         * Localstorage shim from Modernizr
+         *
+         * @method _checkStorage
+         * @private
+         */
+        _checkStorage: function () {
```
Comment (vxsx) [inline static/js/addons/cl.utils.js]: does it make sense to make this self execute so it's only executed once when the page load and not every time we try to access storage?

Comment (FinalAngel) [inline static/js/addons/cl.utils.js]: remember that we tried this and failed :(

Comment (vxsx) [inline static/js/addons/cl.utils.js]:
```js
_isStorageSupported: (function localStorageCheck () {
    var mod = 'modernizr';
    try {
        localStorage.setItem(mod, mod);
        localStorage.removeItem(mod);
        return true;
    } catch (e) {
        return false;
    }
}()),
```
and then
```
getStorage: function (token) {
    if (token && this.isStorageSupported) {
        return localStorage.getItem(token)
    }
}
```
etc

File: static/js/addons/cl.utils.js [hunk 2]
```diff
@@ -95,10 +100,29 @@ var Cl = window.Cl || {};
...
+        _isStorageSupported: (function localStorageCheck() {
+            var mod = 'modernizr';
+            try {
+                localStorage.setItem(mod, mod);
+                localStorage.removeItem(mod);
+                return true;
+            } catch (e) {
+                return false;
+            }
+}()),
```
Comment (vxsx) [inline static/js/addons/cl.utils.js]: indent

**Comment by intellectronica** [comment]
:+1:

**Comment by Brankub** [comment]
:+1:

**Pull request merged**